web/packages/plugins/analysis-platform/src/modules/Analysis/components/CreateAnalysisButton.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Link, useLocation, useParams } from 'react-router-dom'
+import { Button } from '@equinor/eds-core-react'
+
+import { CreateAnalysisButtonProps } from '../../../Types'
+
+export const CreateAnalysisButton = (props: CreateAnalysisButtonProps) => {
+  const { urlPath } = props
+  const location = useLocation()
+  const { entity_id } = useParams<{
+    entity_id: string
+  }>()
+  const to = {
```

Comment:
maybe change name from "to" to "createNewAnalysisPage" or something.

it makes <Link to={to}> -> <Link to={createNewAnalysisPage}> a bit more readable